### ISSUE TITLE
Close issues would not update Gus work item status

### ISSUE THREAD

**Issue by yungcheng** [forcedotcom/git2gus#152]
See this thread entry for more details: 
https://salesforce-internal.slack.com/archives/CKNH4KL3B/p1675803886355649?thread_ts=1675793573.473109&cid=CKNH4KL3B

When GitHub issues are closed, the Gus work item status should be updated accordingly. However the integration does not seem to work at the moment. Using this issue to track that.

**Comment by subakva** [comment]
If this is fixed, I would request the ability to disable it. It is quite common for us to need to change multiple repositories to address a single work item. Having the work item close when part of the fix is shipped would be counter-productive.

**Comment by subakva** [comment]
@mpsenn Not really. Since we're working with multiple git repos, the merging of any given PR doesn't necessarily imply any status change for a work item.

But, we don't currently use the "Integrate" state at all, so maybe we could map that to "In Progress" on our board. It's a little hacky, but it might be a tolerable workaround if we can't disable the feature entirely.
